include/common/test.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef TEST_H
+#define TEST_H
+
+#if TEST
+void run_tests(void);
```

Comment:
If `make format` doesn't remove it, it would be nice to have a blank line between `#if test` and the prototype for symmetry

Reply:
Make format does not remove that blank line. A blank line was added